content/create-an-addon/react/fr/add-to-catalog.md

```diff
@@ -0,0 +1,117 @@
+---
+title: "Ajouter au catalogue d'addon"
+tocTitle: 'Ajouter au catalogue'
+description: 'Partagez votre addon Storybook à la communauté'
+commit: '927e729'
+---
+
+Le [catalogue d'addon](https://storybook.js.org/addons) abrite tous les addons Storybook. C'est là que nous affichons vos addons et où les développeurs en découvrent de nouveaux. Préparons votre addon pour entreprendre sa release, l'empaqueter et le publier dans le catalogue.
+
+![](../../images/catalog.png)
+
+## Préparer votre addon pour le release
```

Comment:
J'y ai pensé à vrai dire, mais dans [la ligne juste au dessus](https://github.com/chromaui/learnstorybook.com/blame/master/content/create-an-addon/react/en/add-to-catalog.md#L8), il y a une distinction entre "release" et "publier". Peut-être qu'on peut remplacer la traduction de cette ligne 8 par "Préparons votre addon pour l'empaqueter et le publier sur npm et dans le catalogue." : ça nous permet d'éviter la répétition tout en gardant cette idée de "release" et de "publish" ?

Reply:
@jonniebigodes We'll probably need your insight here.

The english word "release" is commonly used in french between IT-actors, especially when used as a noun. 
However, things get quite intricate when used as a verb, especially given the [above sentence](https://github.com/chromaui/learnstorybook.com/blame/master/content/create-an-addon/react/en/add-to-catalog.md#L8) since both "release" and "publish" would translate to "publier".

I suggested tweaking that sentence a bit to avoid any weird repetition; this would state "Let's prepare your addon to package it up, and publish it to npm and to the catalog.", with:
* publish to npm = release
* publish to the catalog = (unchanged from original string)

Is it misleading or undesirable in any way ?

Reply:
@Dschungelabenteuer go ahead with the translation on your end. I'll adjust accordingly and add a pull request later on addressing this. The same situation is more than bound to happen with other languages as well. 

Thanks for the heads up on this.